src/platform/Zephyr/GenericBLEManagerImpl_Zephyr.cpp

```diff
@@ -495,6 +498,16 @@ CHIP_ERROR GenericBLEManagerImpl_Zephyr<ImplClass>::HandleTXComplete(const ChipD
     return CHIP_NO_ERROR;
 }
 
+template <class ImplClass>
+void GenericBLEManagerImpl_Zephyr<ImplClass>::HandleBLEAdvertisementTimeout(System::Layer * layer, void * param, System::Error error)
+{
+    if (ConnectivityMgr().IsBLEAdvertisingEnabled())
```

Comment:
If you cancel the timer in StopAdvertising, the check won't be needed anymore.

Reply:
Done